api/get-started/extension-anatomy.md

```diff
@@ -97,7 +97,7 @@ Each VS Code extension must have a `package.json` as its [Extension Manifest](/a
 
 ## Extension Entry File
 
-The extension entry file exports two functions, `activate` and `deactivate`. `activate` is executed when your registered **Activation Event** happens. `deactivate` gives you a chance to clean up before your extension becomes deactivated.
+The extension entry file exports two functions, `activate` and `deactivate`. `activate` is executed when your registered **Activation Event** happens. `deactivate` gives you a chance to clean up before your extension becomes deactivated. For many extensions, explicit cleanup may not be required, and the `deactivate` method can be removed. But if an extension needs to perform an operation when VS Code is shutting down or the extension is disabled or uninstalled, this is the method to do so.
```

Comment:
Change "But" in last sentence to "However, "

Reply:
Thanks for the feedback! Made the change.